src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -101,8 +110,20 @@ using namespace chip::app::Clusters;
 
 {{#if isWritableAttribute}}
 {{#*inline "callbackName"}}DefaultSuccess{{/inline}}
-- (void)write{{>attribute}}WithValue:({{asObjectiveCType type parent.name}})value completionHandler:(StatusCompletion)completionHandler
+- (void)write{{>attribute}}WithValue:({{asObjectiveCType type parent.name}})value completionHandler:(StatusCompletion)completionHandler{
+  CHIPWriteParams * params;
+  {{#if mustUseTimedWrite}}
+  params = [CHIPWriteParams new];
+  params.timedWrite = @10000;
+  {{/if}}
+  [self write{{>attribute}}WithValue:({{asObjectiveCType type parent.name}})value params:params completionHandler:completionHandler];
+}
+- (void)write{{>attribute}}WithValue:({{asObjectiveCType type parent.name}})value params:(CHIPWriteParams * _Nullable)params completionHandler:(StatusCompletion)completionHandler
 {
+    chip::Optional<uint16_t> timedWriteOptional;
+    if (params) {
+      timedWriteOptional.SetValue(params.timedWrite.unsignedShortValue);
```

Comment:
What does this do if `params.timedWrite` is nil?

Reply:
Updated. Thank you.